src/languages/de-DE.js

```diff
@@ -162,7 +162,7 @@ export default {
     Purple: "Lila",
     Pink: "Pink",
     "Search...": "Suchen...",
-    "Heartbeat Retry Interval": "Heartbeat-Wiederholungsintervall",
+    "Heartbeat Retry Interval": "Prüfzeitpunkt-Wiederholungsintervall",
```

Comment:
Phew. I am not sure if we should make this change. Heartbeat is a common term in the IT industry. "Prüfzeitpunkt" reads very cumbersome. But I don't have a better idea right now either. See also [Heartbeat (Informatik)](https://de.wikipedia.org/wiki/Heartbeat_(Informatik))

Reply:
Thanks for your effort, I am aware of the "heartbeat" term, nevertheless i chose to use the more "familiar" word.
Anyway, I thank you for your effort and will accommodate your changes.

Reply:
How about `Intervall der Statusabfrage` or `Prüfungsintervall`

Reply:
> How about `Intervall der Statusabfrage` or `Prüfungsintervall`

`Prüfintervall` or `Überprüfungsintervall` come to mind. `Prüfung` surely is a german noun for `to check` but has more of an examination character.

Reply:
`Überprüfungsintervall` is surely the best choice here.

Reply:
I implemented your proposed changes, tyvm.